README.md

```diff
@@ -35,16 +35,16 @@ You can also download it from [here](https://github.com/qbittorrent/qBittorrent/
 
 ### Misc:
 For more information please visit:
-http://www.qbittorrent.org
+https://www.qbittorrent.org
```

Comment:
Have you tried open the links?
Only this URL works.

Reply:
At the time, yes, they all loaded fine.

Not sure why they removed https support now. And the forum url has changed completely.

Oh well.